provisioning/tasks/drush-shield.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Enable Shield module
+  command: >
+    {{ drush_path }} pm-enable -y shield
+    chdir={{ drupal_core_path }}
+  when: "{{ drupal_shield_enabled }} == true"
+  sudo: no
+
+- name: Enable Shield module configuration
+  command: >
+    {{ drush_path }} variable-set shield_enabled 1 -y
+    chdir={{ drupal_core_path }}
+  when: "{{ drupal_shield_enabled }} == true"
+  sudo: no
+
+- name: Disable Shield module configuration
```

Comment:
Other than that, I have tested the configuration change and it works. So happy to merge this when this is discussed.